core/node/libp2p/host.go

```diff
@@ -43,9 +46,21 @@ func Host(mctx helpers.MetricsCtx, lc fx.Lifecycle, params P2PHostIn) (out P2PHo
 	}
 
 	ctx := helpers.LifecycleCtx(mctx, lc)
+	cfg, err := params.Repo.Config()
+	if err != nil {
+		return out, err
+	}
+	var bootstrappers []ma.Multiaddr
+	for _, addrStr := range cfg.Bootstrap {
```

Comment:
@Stebalien Is the default for cfg.Bootstrap the DefaultBootstrapPeers in go-libp2p-kad-dht ?

Reply:
Basically, yes. Although I think it's duplicated.